README.md

```diff
@@ -0,0 +1,81 @@
+# SSS TODO-track-name
+
+This is the private (internal) repository for the **TODO-track-name** track of the Security Summer School (SSS).
+It contains deployment files and scripts, source code files for activities and CTF challenges.
+This is to be used internally by members of the SSS **TODO-track-name** team for content development.
+It can be made available to partners who want to use the content for their own class / event.
+
+Public content is available to students, participants and hobbyists in the [public repository pair](https://github.com/security-summer-school/template-public/) (TODO: fix link).
+
+## Developing Content
+
+Content is to be developed both in this private (internal) repository and in the [public repository pair](https://github.com/security-summer-school/template-public/) (TODO: fix link).
+See [the contribution guide](CONTRIBUTING.md) on how you could report or fix issues and on how you can improve content.
+
+Reviewers are requested to follow [the reviewing guide](REVIEWING.md).
+
+For in-depth discussions, please join the [SSS Discord community](https://discord.gg/qx3RmNEVkB).
+
+## CTF Challenges
+
+CTF (*Capture the Flag*) challenges are only located in this private repository.
+These are challenges that are to be solved by participants in simulated contents.
+Participants will only be provided with a link to a CTF platform that hosts challenges: description, support files, form for submitting the flag.
+
+The `ctf/` folder stores CTF challenges.
+There are typically two CTFs: a `final-ctf` and a `mid-ctf`.
+Each challenge has its own folder.
+
+### CTF Challenge Folder Structure
+
+A CTF challenge folder consists of:
+
+* `README.md`: details the challenge name, challenge description, points, hints, vulnerability overview, solution overview, deployment instructions
+* `flag`: contains the flag string
+* `sol/`: stores the reference solution scripts (`solution.py`, `solution.sh`, `solution.c` + `Makefile` or others), and a solution write-up (a detailed set of steps) in `README.md`.
+* `src/` (optional): stores source code and build scripts to generate deployment or public files.
+  To keep things generic, flags shouldn't be hard-coded in source code files.
+  The recommended way is to use a template file and then use a generator script (using `sed` or something similar) that generates the actual file (with the flag inside).
+
+  Most challenges will require a source code folder, but some may not, making this optional.
+* `public/` (optional): stores public files that are to be packed in an archive that is to be made available to participants.
+  They are usually generated from src/ folder, though they may also be static files.
+
```

Comment:
Extra line?

Reply:
Yes, to force a newline in the rendering of Markdown content.